src/server/scripts/Northrend/FrozenHalls/HallsOfReflection/fix instance_halls_of_reflection.cpp

```diff
@@ -230,11 +231,12 @@ class instance_halls_of_reflection : public InstanceMapScript
             {
                 case NPC_SYLVANAS_PART1:
                     creature->SetVisible(false);
+                    creature->SetSpeed(MOVE_RUN, 1.1);
                     NPC_LeaderIntroGUID = creature->GetGUID();
                     if (TeamIdInInstance == TEAM_ALLIANCE)
                         creature->UpdateEntry(NPC_JAINA_PART1);
                     creature->RemoveNpcFlag(UNIT_NPC_FLAG_GOSSIP | UNIT_NPC_FLAG_QUESTGIVER);
-                    creature->RemoveUnitMovementFlag(MOVEMENTFLAG_WALKING);
+                    creature->RemoveUnitMovementFlag(MOVEMENTFLAG_WALKING);                 
```

Comment:
remove the whitespaces added at the end of the line

Reply:
> remove the whitespaces added at the end of the line

Thanks for the info, I didn't know :)